src/ImageProcessorCore/Formats/Tiff/ValueDecoders/TiffIptcValueDecoder.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using ImageProcessorCore.IO;
+
+namespace ImageProcessorCore.Formats.Tiff.ValueDecoders
+{
+    internal class TiffIptcValueDecoder : ITiffValueDecoder
+    {
+        public bool DecodeValue(TiffReader reader, TiffProperty value, int count)
+        {
+            if( value.Tag.TagId != 33723)
+                return false;
+
+            IptcDecoder decoder = IptcDecoder.Create(reader.BaseStream, count*4);
```

Comment:
Let's throw an `ImageFormatException` for now and see how many errors we get. There's a lot of error handling in the `JpegDecoder` already anyway.